handler.go

```diff
@@ -566,38 +566,40 @@ func writeIgnResourceVersionHeader(w http.ResponseWriter, version int) {
 	w.Header().Set("X-Ign-Resource-Version", strconv.Itoa(version))
 }
 
-// serveFileOrLink returns the link where the client can download the zip file from when linkRequested is set to true.
-// If set to false, it will stream the file from the host machine directly to the client.
+// serveFileOrLink returns a link to download the provided zip file from if linkRequested is set to true.
+// If linkRequested is set to false, it will stream the file from the host machine directly to the client.
 //
-//	link must contain the URL where to download the resource from when linkRequested is set to true or,
-//	link must contain the path to the zip file when linkRequested is set to false.
+//	If linkRequested is set to true, link must contain the URL where to download the resource from when linkRequested is set to true or,
+//	if linkRequested is set to false, link must contain the path to the zip file when linkRequested is set to false.
```

Comment:
Repeating the `if linkRequested is set to false` has a strange cadence.

We can use `if linkRequested is set to true/false` as sort of titles and then list the things that happen and the requirements is has for each case.

```
If `linkRequested` is set to true: 
    [This function will...]
    `link` must contain the URL where to download the resource from when `linkRequested` is set to true
If `linkRequested` is set to false:
    This function will stream the file from the host machine directly to the client.
    `link` must contain the path to the zip file when `linkRequested` is set to false.
```